packages/web3-wallet/src/hd-wallet.ts

```diff
@@ -98,7 +98,7 @@ export function getHDWalletPath(keyType: KeyType, addressIndex: number): string
   }
   // Being explicit: we always use coinType 1234 no matter the network.
   const coinType = "1234'"
-  const keyTypeNum = (keyType === 'default' || keyType === 'gl-secp256k1') ? 0 : 1
+  const keyTypeNum = keyType === 'default' || keyType === 'gl-secp256k1' ? 0 : 1
```

Comment:
Why do we need to remove the parenthesis?

Reply:
It was removed by `npm run lint:fix`.